core/cas-server-core-tickets/src/test/java/org/apereo/cas/ticket/registry/DefaultTicketRegistryTests.java

```diff
@@ -77,8 +85,28 @@ void verifyCountForService() throws Throwable {
     void verifyEncodeFails() throws Throwable {
         val cipher = new DefaultTicketCipherExecutor(null, null,
             "AES", 512, 16, "webflow");
-        val reg = new DefaultTicketRegistry(cipher, mock(TicketSerializationManager.class), new DefaultTicketCatalog());
+        val reg = new DefaultTicketRegistry(cipher, mock(TicketSerializationManager.class), new DefaultTicketCatalog(),
+                mock(ConfigurableApplicationContext.class));
         assertNull(reg.encodeTicket(null));
         assertNotNull(reg.decodeTicket(mock(Ticket.class)));
     }
+
+    @RepeatedTest(1)
+    void verifyGetExpiredTicket() throws Throwable {
+        val applicationContext = mock(ConfigurableApplicationContext.class);
+        val registry = new DefaultTicketRegistry(mock(TicketSerializationManager.class), new DefaultTicketCatalog(),
+                applicationContext);
```

Comment:
Should call `getNewTicketRegistry`

Reply:
In fact, no as we use a specific `applicationContext` here, on which we make checks at the end of the test.